Scripts/script-FindSimilarIncidentsByText.yml

```diff
@@ -8,16 +8,16 @@ script: |-
   TEXT_FIELDS = set(map(lambda x: x.lower(), demisto.args()['textFields'].split(',')))
```

Comment:
all the changes in this file won't break backward?

Reply:
no.. changed const in the code to arguments with the same default value as the consts